examples/jsm/renderers/webgpu/WebGPUBackend.js

```diff
@@ -1,3 +1,7 @@
+/*// debugger tools
+import 'https://greggman.github.io/webgpu-avoid-redundant-state-setting/webgpu-check-redundant-state-setting.js';
```

Comment:
@greggman I've added its lib here, so it can inspire other developers to build good tools for it :)

Reply:
Maybe we can add some debug toggle (for example, global variable like `window.DEBUG`), which will dynamic-import the lib if enabled?

Reply:
Or maybe `THREE.Renderer( { debug: true } )`

Reply:
Yeah... And maybe place `renderer.info` also under that toggle? Like, it isn't updating (and there are no such property) when it is disabled.

Reply:
> Yeah... And maybe place renderer.info also under that toggle? Like, it isn't updating (and there is no such property) when it is disabled.

I think `debug` would be a property for development, `info` can still be useful for inspection.